src/components/SearchableMultiSelect/SearchableMultiSelect.jsx

```diff
@@ -580,6 +595,7 @@ const SearchableMultiSelect = createClass({
 								'&-search',
 								{
 									'&-search-is-small': isSmall,
+									'&-search-is-error': errorChildProps && errorChildProps.children && errorChildProps.children !== true
```

Comment:
I'm a little confused by the logic here. Suppose I do pass `Error={true}`, wouldn't that result in this expression being `false`?

Reply:
good find, I re-used some of the logic for the error-content div logic and forgot to update it for the case of no text. got it updated now

Reply:
I don't think we need to do anything with them. The only thing we care about is the `children` which you're already using. I just took a look at `Validation` and it claims to have a `description` prop but I think that's wrong. The only thing it seems to care about is `children`.

Reply:
right, the error border should be applied if there are children but if the child is true then we know there's a general validation applied to the component. thus removing that last piece of logic will allow passing `Error={true}`  to validate the search field but not have a message. I added a test case to check this, will push up shortly :)